### PR TITLE
Task #295 changed the global init to a more robust init all variables. I think …

### PR DESCRIPTION
**Pull request by awbrown90**
…most students were having problems because the global init is a new addition to tensorflow v12. I tried using tf v12 and

while the global init may have worked there were other issues such as using the import data_input. Using the init all variables command works well though and I think it will be good for
most students setups. The init is not needed in the final code block for running test accuracy if it is already excuted during the training code block.

### CODE REVIEW

**Comment by awbrown90** [comment]
closing pull request as init all variables is deprecated and will discontinue being used past 2017/3/02
